forward.ml

```diff
@@ -18,7 +18,6 @@ open Options
 open Ast
 open Types
 open Atom
-open Pervasives
```

Comment:
Did the compiler tell you there was no use of `Stdlib` here? If not removing an open is dangerous as it shadows comparision functions such as `compare` that might be used somewhere